lighthouse-core/lib/i18n/swap-locale.js

```diff
@@ -50,7 +50,8 @@ function swapLocale(lhr, requestedLocale) {
 
   const locale = i18n.lookupLocale(requestedLocale);
   const {icuMessagePaths} = lhr.i18n;
-  const missingIcuMessageIds = /** @type {string[]} */([]);
+  /** @type {string[]} */
```

Comment:
does this have any practical effect?

Reply:
> does this have any practical effect?

no, just a cast is overkill and may cause a double take :)